reactpy_router/core.py

```diff
@@ -113,9 +117,9 @@ def _match_route(
     return None
 
 
-_link = export(
+_link, _history = export(
```

Comment:
Need to create an issue detailing our long term solution for back-button behavior

Reply:
That's here: https://github.com/reactive-python/reactpy-router/issues/19